roslibrust/src/ros1/tcpros.rs

```diff
@@ -118,3 +117,44 @@ impl ConnectionHeader {
         Ok(header_data)
     }
 }
+
+pub async fn establish_connection(
+    node_name: &Name,
+    topic_name: &str,
+    server_uri: &str,
+    conn_header: ConnectionHeader,
+) -> Result<TcpStream, std::io::Error> {
+    use tokio::io::{AsyncReadExt, AsyncWriteExt};
```

Comment:
I had to add the `use` statement for these here since they conflict with `std::io::{Read, Write}`.